CMakeLists.txt

```diff
@@ -0,0 +1,201 @@
+cmake_minimum_required(VERSION 3.8)
+
+project(usd-unity-sdk LANGUAGES CXX)
+set (CMAKE_CXX_STANDARD 11)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
+
+find_package(pxr REQUIRED)
+find_package(SWIG 3.0.12 REQUIRED)
+find_package(Boost REQUIRED)
+find_package(Python COMPONENTS Development)
+find_package(TBB REQUIRED)
+find_package(Unity REQUIRED)
```

Comment:
@cpredmann do you need both Unity and a C# compiler ?

Reply:
The FindUnity.cmake module also gets the C# compiler (in addition to Unity), and it does need both, since the USD.NET.Unity .dll references UnityEngine.dll.

Reply:
Might it make sense to add the USD.NET.Unity dll as C# files in the package and let Unity compile them every time ?

Reply:
I'd lean towards just compiling the dll at build, since the overwhelming majority are just Swig generated and don't make much sense out of that context

Reply:
Although that's for the USD.NET.dll, so through C# for the .Unity dll could be packaged as is

Reply:
basically what I am getting at is that requiring a unity installation makes for a very CI unfriendly setup, if ever you want to automate the compilation of USD and/or dependencies.

Reply:
Got it... so in that case compiling the Swig generated C# only requires a mono compiler, so it may make sense to build the USD.NET.dll, but not the USD.NET.Unity.dll (which is the only thing requiring a Unity install)